content/doc/administration/requirements/linux.adoc

```diff
@@ -0,0 +1,78 @@
+---
+layout: documentation
+title:  Linux Support Policy
+---
+
+This page documents the Linux support policy for the Jenkins controller and agents.
+
+== Scope
+
+Jenkins plugins may set additional requirements to Linux versions on controllers and/or agents.
+This page does not document such requirements.
+Please refer to plugin documentation.
+
+== Why?
+
+Theoretically, Jenkins can run everywhere where you can run a supported Java version,
+but there are some limitations in practice.
+Jenkins core and some plugins include native code or depend on Linux API and subsystems,
+and hence they rely on specific Linux versions.
+Jenkins platform specific installation packages rely on specific Linux versions.
+
+== Support levels
+
+We define multiple support levels for Linux platforms.
+
+[width="100%",cols="20%,35%,45%",options="header",]
+|===
+|Support level |Description |Platforms
+
+| **Level 1** - Full support
+| We run automated testing for these platforms, and we intend to fix the reported issues timely.
+a|
+  * 64-bit (amd64) Linux versions supported by Debian, Red Hat, SUSE, and Ubuntu
+  * Linux versions used in the official Docker images
+
+| **Level 2** - Supported
+| We do not actively test these platforms, but we intend to keep compatibility.
+  We will consider patches if they do not put Level 1 support at risk and do not create maintenance overhead.
+a|
+  * 64-bit (amd64, arm64, s390x, ppc64le) Linux versions that use the Debian packaging format
+  * 64-bit (amd64, arm64, s390x, ppc64le) Linux versions that use the rpm packaging format
+
+| **Level 3** - Patches considered
+| Support may have limitations and extra requirements.
+  We do not test compatibility, and we may drop support if there is a need.
+  We will consider patches if they do not put level 1/2 support at risk and do not create maintenance overhead.
+a|
+  * 32-bit (x86, arm) Linux versions
+  * RISC-V and other architectures not included in level 1 or level 2 support
+  * Preview releases
+
+| **Level 4** - Unsupported
+| These versions are known to be incompatible or to have severe limitations.
+  We do not support the listed platforms, and we will not accept patches.
+a|
+  * Linux versions no longer supported by operating system providers
+|===
+
+== References
+
+* link:https://wiki.debian.org/LTS[Debian Long Term Support]
+* link:https://access.redhat.com/support/policy/updates/errata[Red Hat Enterprise Linux Life Cycle]
+* link:https://www.suse.com/lifecycle/[SUSE Product Support Lifecycle]
+* link:https://ubuntu.com/about/release-cycle[Ubuntu lifecycle and release cadence]
+
+== Contributing
+
+If you would like to add support for more Linux platforms or to share feedback,
+we will appreciate your contributions!
```

Comment:
```suggestion
You are welcome to propose PR's that add support for other Linux platforms or to share feedback;
we will appreciate your contributions!
```
I'm not sure if PR's is the right noun to use here.  Packages, software, ...?  But just "add support" seems ambiguous -- anyone can add support to their Jenkins installation.  I think we are talking about what they contribute back to the produce, and there they can propose software that the community does not accept.